### DISCUSSION TITLE
Setting up several OIDC clients

### DISCUSSION THREAD

**Discussion by DenisIlyushin** [outline/outline#4692] [Self Hosting]
Is there a way to set up more than one OIDC client in outline and how to cofing docker-compose.yaml for that?

**Comment by DenisIlyushin** [comment]
I mean, is there a way to add different login options on the login screen (add other buttons) for self-hosted solution?

**Reply by tommoor** [reply]
It's only possible to configure a single authentication provider of each type, it would be quite an extensive refactor to enable multiple. What's your usecase?

**Reply by DenisIlyushin** [reply]
I've asked to create outline wiki that will be used by different teams, each team has separate identity.
Like, i have developers that have gitlab accounts and managers that use LDAP accounts for login.

**Reply by DenisIlyushin** [reply]
I was trying to cofigure two keycloak providers, one that use openid-connect and other - SAML

**Reply by tommoor** [reply]
You could achieve this with a license for the enterprise edition which supports SAML as a separate provider to OIDC.
